src/core/blueprint/into_schema.rs

```diff
@@ -123,6 +123,10 @@ fn to_type(def: &Definition) -> dynamic::Type {
                 if let Some(description) = &field.description {
                     input_field = input_field.description(description);
                 }
+                if let Some(default_value) = field.default_value.clone() {
+                    input_field =
+                        input_field.default_value(ConstValue::from_json(default_value).unwrap())
```

Comment:
Drop unwrap.

Reply:
use `trace::warning` to show that transformation failed over here.

Reply:
done